src/camvidlog2/cli.py

```
@@ -95,37 +99,41 @@ def query(
+            except FrameError:
+                continue
+            outpath = outdir / "avg" / f"{result.rank:03d}.jpg"
+            save(outpath, img_array)
+            del img_array
+    # TODO output csv
+
+    # output for each query separately
+    for j, _ in enumerate(embedding_group.items, 0):
+        for result in calculate_results(distances[j], num, roll):
+            print(
+                f"{j + 1:3d} {result.rank:3d} {result.frame_no:4d} {result.score:.3f} {result.filename}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)